src/main/java/com.outsystems.plugins.inappbrowser/osinappbrowserlib/views/OSIABWebViewActivity.kt

```diff
@@ -440,7 +443,7 @@ class OSIABWebViewActivity : AppCompatActivity() {
             this@OSIABWebViewActivity.filePathCallback = filePathCallback
             val intent = fileChooserParams?.createIntent()
             try {
-                fileChooserLauncher.launch(intent)
+                fileChooserLauncher.launch(intent!!)
```

Comment:
Can we avoid force unwrapping here? This could lead to issues.

Reply:
The reason I did the `!!` is because it's inside a try catch, so if by chance the `intent` is null (because `fileChooserParams` is null), the NPE is caught and `false` is returned

What's preferable here? to check if `intent` is null and return true instead?

Reply:
Yes, that's fine, but why didn't we need the `!!` before and we do now?

Reply:
Ah that's because when I added the `activity-ktx` dependency, it basically updated androidx's activity version, and now it gives a compile error if trying to send a null intent to `ActivityResultLauncher#launch`.

Reply:
I see, I knew something had to change ahah. In that case, I think we're fine since as you said we're doing that inside the try-catch block. Nonetheless, can we catch the `NullPointerException` it its own catch block and Log the according error?

Reply:
Done in 568b6a0ccf2c7168e432baca2e05783bc4dc5e6a